tools/ManifestSignerClient/ManifestSignerClient/JsonCanonization/NumberCachedPowers.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2010 the V8 project authors. All rights reserved.
```

Comment:
Do we know what license of this code opensourced under? Can we use it commercially?